Dockerfile

```diff
@@ -50,6 +50,8 @@ mamba install -y -n base \
     "rapids=${RAPIDS_VER}.*" \
     "dask-sql=${DASK_SQL_VER%.*}.*" \
     "python=${PYTHON_VER}.*" \
+    # cuda-python pin required for CUDA 12. See https://github.com/conda-forge/cuda-python-feedstock/issues/66
```

Comment:
Caution: I don't think you can put a comment in the middle of a bash line continuation `\`. That might be why CI is failing (I didn't look at the logs).

Reply:
Yeah, I thought it worked because I ran it locally but not long enough for it to fail... I pushed a fix.